react-components/src/hooks/useSceneDefaultCamera.tsx

```diff
@@ -26,26 +26,34 @@ export const useSceneDefaultCamera = (
       data.sceneConfiguration.cameraTranslationZ
     );
 
-    const rotation = new Quaternion().setFromEuler(
-      new Euler(
-        MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationX),
-        MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationY),
-        MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationZ),
-        'XYZ'
-      )
-    );
-
-    // As a heuristic, use distance to center of all models' bounding
-    // boxes as target distance
-    const positionToSceneCenterDistance = position.distanceTo(
-      viewer.models
-        .reduce((acc, m) => acc.union(m.getModelBoundingBox()), new Box3())
-        .getCenter(new Vector3())
-    );
-    const target = position
-      .clone()
-      .add(new Vector3(0, 0, -positionToSceneCenterDistance).applyQuaternion(rotation));
+    let target: Vector3;
+    if (data.sceneConfiguration.cameraTargetX !== undefined) {
+      target = new Vector3(
+        data.sceneConfiguration.cameraTargetX,
+        data.sceneConfiguration.cameraTargetY,
+        data.sceneConfiguration.cameraTargetZ
+      );
+    } else {
+      const rotation = new Quaternion().setFromEuler(
+        new Euler(
+          MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationX),
+          MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationY),
+          MathUtils.degToRad(data.sceneConfiguration.cameraEulerRotationZ),
+          'XYZ'
+        )
+      );
 
+      // As a heuristic, use distance to center of all models' bounding
+      // boxes as target distance
+      const positionToSceneCenterDistance = position.distanceTo(
+        viewer.models
+          .reduce((acc, m) => acc.union(m.getModelBoundingBox()), new Box3())
+          .getCenter(new Vector3())
+      );
+      target = position
+        .clone()
+        .add(new Vector3(0, 0, -positionToSceneCenterDistance).applyQuaternion(rotation));
+    }
```

Comment:
Put this in a function, e.g. `extractCameraTarget`

Reply:
Fixed in [8c26ebf](https://github.com/cognitedata/reveal/pull/4221/commits/8c26ebf8c0971db9c2b4ce288dba58769370f3a8)